api/src/test/java/uk/gov/hmcts/payment/api/controllers/PaymentControllerTest.java

```diff
@@ -385,10 +386,9 @@ public void searchAllPayments_withCcdCaseNumber_shouldReturnRequiredFieldsForVis
 
         assertThat(payment.getCcdCaseNumber()).isEqualTo("ccdCaseNumber1");
 
-        assertThat(payment.getReference()).isNotBlank();
+        assertThat(payment.getPaymentReference() == null || !payment.getPaymentReference().trim().isEmpty()).isTrue();
```

Comment:
isNotBlank() behaviour has changed, it used to be true when null but after the update it's false when null. https://github.com/assertj/assertj/issues/1069

Reply:
isfalse is easier to understand then not istrue